rclpy/rclpy/__init__.py

```diff
@@ -109,6 +110,7 @@ def spin_once(node, *, timeout_sec=None):
         if request:
             response = srv.callback(request, srv.srv_type.Response())
             srv.send_response(response, header)
+    gc.collect()
```

Comment:
I tried doing that, but it did not seem to have any effect. Is there another way of achieving the same effect?